src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -662,115 +344,79 @@ def post(self):
-        #         bounds_path = Path(workdir).resolve().parent
-        #         bounds_path = os.path.join(bounds_path.__str__() + "bounds.json")
-        # with open(bounds_path, "w") as file:
-        #     json.dump(bounds_dict, fp=file)
-
-        root = self._get_runinfo_path_root()
-        results_path = root.findtext("pi:outputResultsFile", namespaces=ns)
-        results_dict = dict()
-
-        for key, values in results.items():
-            new_value = values.tolist()
-            if len(new_value) == 1:
-                new_value = new_value[0]
-            results_dict[key] = new_value
-
-        results_path = os.path.join(workdir, "results.json")
-        if results_path is None:
-            workdir = root.findtext("pi:workDir", namespaces=ns)
-            results_path = os.path.join(workdir, "results.json")
-            if not Path(workdir).is_absolute():
-                results_path = Path(workdir).resolve().parent
-                results_path = os.path.join(results_path.__str__() + "results.json")
-        with open(results_path, "w") as file:
-            json.dump(results_dict, fp=file)
+        if os.path.exists(self.output_folder) and self._save_json:
+            self._write_json_output()
 
 
-def connect_database():
-    client = InfluxDBClient(
-        host=DB_HOST, port=DB_PORT, username=DB_USER, password=DB_PASSWORD, database=DB_NAME
-    )
-    if DB_NAME not in client.get_list_database():
-        client.create_database(DB_NAME)
-    return client
+class EndScenarioSizingHIGHS(EndScenarioSizing):
+    """
+    HIGHS is now the standard solver and gurobi only to be used when called specifically.
+    Currently, the classes in HIGHS are maintained such that the same 'old' function calling can be
+    used for the code running in NWN.
+    """
 
+    pass
 
-def format_datetime(dt):
-    date, time = dt.split(" ")
-    day, month, year = date.split("-")
-    ndate = year + "-" + month + "-" + day
-    ntime = time + ":00+0000"
-    return ndate + "T" + ntime
 
+class EndScenarioSizingGurobi(SolverGurobi, EndScenarioSizing):
+    """
+    Uses Gurobi as the solver for the EndScenarioSizing problem.
+    """
 
-class EndScenarioSizingHIGHS(EndScenarioSizing):
-    def post(self):
-        super().post()
+    pass
 
-        self._write_updated_esdl(self.get_energy_system_copy())
 
-    def solver_options(self):
-        options = super().solver_options()
-        options["casadi_solver"] = self._qpsol
-        options["solver"] = "highs"
-        highs_options = options["highs"] = {}
-        highs_options["mip_rel_gap"] = 0.02
+class EndScenarioSizingDiscounted(EndScenarioSizing):
+    """
+    The discounted annualized is utilised as the objective function.
+    The change of the objective function is done by changing the option 'discounted_annulized_cost'
+    to True
 
-        options["gurobi"] = None
+    Goal priorities are:
+    1. Match heat demand with target
+    2. minimize TCO = Capex + Opex*lifetime
```

Comment:
I think we should say explicitly say that we use the technical lifetime to compute an annualized discounted capex and thus we also use allow for assets with varying depreciation times